.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: java
+sudo: false # faster builds
+
+script: "cd EbxLinkedIn && mvn clean package -Dmaven.test.failure.ignore=true"
```

Comment:
are we sure we want to ignore test failures? 

Please also see https://stackoverflow.com/questions/19684421/how-to-get-surefire-reports-form-travis-ci-build to ensure we get the surefire reports if there are test failures.